funcs/func.py

```diff
@@ -1,2 +1,12 @@
 def sum(x: float, y: float) -> float:
-    return x + y
\ No newline at end of file
+    return x + y
+
+
+# def misformatted_func(
+# var1, var2, var3
+# ):
+#     print(var1, var2, var3)
+
+
+def corr_formatted_func(var1, var2, var3):
```

Comment:
Don't shorten anything! Also don't think this is needed but again good you tested it!

Reply:
Removed.